docs/installation/source.rst

```diff
@@ -92,3 +92,21 @@ active virtualenv environment, run:
 
     cd mango-viz
     make prepare
+
+
+For Mango developers - if the contents of the example notebooks
```

Comment:
Notes for generating docs should not be visible by people reading the docs. This should go somewhere else.

Reply:
done